tests/test_georaster.py

```diff
@@ -213,7 +213,7 @@ def test_interp(self):
 
         xmin, ymin, xmax, ymax = r.ds.bounds
 
-        # testing interp, find_value, and read when it falls right on the coordinates
+        # Test for several points: interp, value_at_coords, and read should be the same right on the coordinates
```

Comment:
It would be good to check that it works fine after the data has been reprojected, cropped or downsampled...

Reply:
Done for crop. This also changes the GDAL interpretation to "Area" instead of "Point" for our test sample. So I tested that this worked correctly as well.